src/subscription/PriceUtils.ts

```diff
@@ -10,6 +10,26 @@ import {UpgradePriceService} from "../api/entities/sys/Services"
 import {getTotalAliases, getTotalStorageCapacity, hasAllFeaturesInPlan, isBusinessFeatureActive, isSharingActive, isWhitelabelActive} from "./SubscriptionUtils"
 import {IServiceExecutor} from "../api/common/ServiceRequest"
 import {ConnectionError} from "../api/common/error/RestError"
+import {ProgrammingError} from "../api/common/error/ProgrammingError.js"
+
+export const enum PaymentInterval {
+	Monthly = 1,
+	Yearly= 12
+}
+
+export function asPaymentInterval(paymentInterval: string | number): PaymentInterval {
+	if (typeof paymentInterval === "string") {
+		paymentInterval = Number(paymentInterval)
+	}
+	switch (paymentInterval) {
+		case Number(PaymentInterval.Monthly):
```

Comment:
why do you convert to number again here?

Reply:
we do this to make it robust against changes to the PaymentInterval enum, e.g. removing the "const" or changing the value to string

Reply:
you shouldn't be able to do `switch` if you change it to string  anyway?